thunder.profile

```diff
@@ -319,22 +319,6 @@ function thunder_modules_installed($modules) {
     }
   }
 
-  // Enable riddle paragraph in field_paragraphs.
-  if (in_array('thunder_riddle', $modules)) {
-
-    /** @var \Drupal\field\Entity\FieldConfig $field */
-    $field = entity_load('field_config', 'node.article.field_paragraphs');
-
-    $settings = $field->getSetting('handler_settings');
-
-    $settings['target_bundles']['riddle'] = 'riddle';
-    $settings['target_bundles_drag_drop']['riddle'] = ['enabled' => TRUE, 'weight' => 10];
-
-    $field->setSetting('handler_settings', $settings);
-
-    $field->save();
-  }
-
```

Comment:
we should add disabling this option just in the integration test, so it does not fail